### ISSUE TITLE
generate new schedule

### ISSUE THREAD

**Issue by bijae** [AlexRiedel/V-PlanMANOS#3]
User request: Build new daily schedule from teaching cover schedule and current schedule (http://manos-dresden.de/schule/stundenplan.php).

**Comment by AlexRiedel** [comment]
Good idea, but in Android it is very difficult to extract information from a *.pdf. I didn't find free  library for this purpose and the commercial ones are at at least 700$ a year.
So, maybe it would be better to implement the normal schedule in an XML-file, or to have these information in a database on a web-server.
(Of course this information have to be filled in manually and must be maintained manually as long as anyone uses the app.)

**Comment by bijae** [comment]
I have taken a look at pdfClown, which is an apache licensed library for java. I will test it and upload a revision as soon as possible.
